cmd/kubeadm/app/phases/addons/proxy/proxy.go

```diff
@@ -39,10 +39,6 @@ import (
 )
 
 const (
-	// KubeProxyClusterRoleName sets the name for the kube-proxy ClusterRole
-	// TODO: This k8s-generic, well-known constant should be fetchable from another source, not be in this package
-	KubeProxyClusterRoleName = "system:node-proxier"
-
```

Comment:
in this file we are leaving some constants, should those be moved to constants.go as well?